src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleConsoleSpammer.kt

```diff
@@ -0,0 +1,70 @@
+package net.ccbluex.liquidbounce.features.module.modules.exploit
+
+import io.netty.buffer.Unpooled
+import net.ccbluex.liquidbounce.config.NamedChoice
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.minecraft.client.render.entity.PlayerModelPart
+import net.minecraft.network.PacketByteBuf
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket
+import net.minecraft.network.packet.c2s.play.CustomPayloadC2SPacket
+import net.minecraft.util.Identifier
+import kotlin.random.Random
+
+object ModuleConsoleSpammer : Module("ConsoleSpammer", Category.EXPLOIT) {
+
+    private val delay by int("Delay", 0, 0..20)
+    private val modes by enumChoice("Mode", Modes.PAYLOAD, Modes.values())
+    private val payload = PacketByteBuf(Unpooled.buffer())
+    private val vulnerableChannels = arrayOf("MC|BEdit", "MC|BSign", "MC|TrSel", "MC|PickItem")
+
+    val repeatable = repeatable {
+        if (mc.world == null) {
```

Comment:
remove those useless null checks because world and player can never be null in a repeatable.